guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc

```diff
@@ -49,6 +49,10 @@ For more information, see the Red Hat Knowledgebase solution https://access.redh
 ifdef::satellite[]
 You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious}.
 
+During the upgrade, the PostgreSQL data is moved from `/var/opt/rh/rh-postgresql12/lib/pgsql/` to `/var/lib/pgsql/`.
+If these two paths reside on the same partition, no further action is required.
+If they reside on different partitions, ensure that there is enough space for the data to be copied over.
```

Comment:
Not sure how to properly phrase this (and forgot to mention this to you when we originally talked about this), but I'd add something like: Alternatively, you can move the data on your own and the upgrade will skip this step if `/var/opt/rh/rh-postgresql12/lib/pgsql/data/` does not exist.